src/components/GlobalPreferences/CustomLabels/LocalIdentities.js

```diff
@@ -409,7 +409,7 @@ const List = styled.ul`
   padding: 0;
   list-style: none;
   overflow: hidden;
-  width: calc(100% + ${5 * GU}px);
+  width: calc(100% + ${6 * GU}px);
```

Comment:
I might need to double check this, because if I recall correctly I decreased the width because this was overflowing the container on mobile.